daily-challenges/2024-11/29/problem.md

```
@@ -0,0 +1,9 @@
+# Playwright challenge
```

Markdownlint (0.35.0)

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)